crates/test_utils/tests/connectors/authorizedotnet_wh_ui.rs

```diff
@@ -0,0 +1,37 @@
+use rand::Rng;
+use serial_test::serial;
+use thirtyfour::{prelude::*, WebDriver};
+
+use crate::{selenium::*, tester};
+
+struct AuthorizedotnetSeleniumTest;
+
+impl SeleniumTest for AuthorizedotnetSeleniumTest {
+    fn get_connector_name(&self) -> String {
+        "authorizedotnet".to_string()
+    }
+}
+
+async fn should_make_webhook(web_driver: WebDriver) -> Result<(), WebDriverError> {
+    let conn = AuthorizedotnetSeleniumTest {};
+    let amount = rand::thread_rng().gen_range(50..1000); //This connector detects it as fradulent payment if the same amount is used for multiple payments so random amount is passed for testing(
+    conn.make_webhook_test(
+        web_driver,
+        &format!("{CHEKOUT_BASE_URL}/saved/227?amount={amount}"),
+        vec![
+            Event::Trigger(Trigger::Click(By::Id("card-submit-btn"))),
+            Event::Assert(Assert::IsPresent("status")),
+            Event::Assert(Assert::IsPresent("processing")),
+        ],
+        10,
+        "processing",
```

Comment:
why are we expecting processing here, it should be a terminal status right?

Reply:
This connector status changes to success in one day, although we get processing webhook from connector end.